src/coprocessor/util.rs

```diff
@@ -134,6 +134,62 @@ pub fn get_pk(col: &ColumnInfo, h: i64) -> Datum {
     }
 }
 
+use std::time::Duration;
+use std::sync::atomic::{AtomicIsize, Ordering};
+
+// RuntimeStats collects one executor's execution info.
+#[derive(Debug)]
+pub struct RuntimeStats {
```

Comment:
Maybe better to integrate with `ExecLocalMetrics`?

Reply:
`ExecLocalMetrics` or `ExecutorMetrics`, I only found `ExecutorMetrics` in `Index_scan`

Reply:
Oh yes, it should be `ExecutorMetrics`. `ExecLocalMetrics` is just another thing.